codec/encoder/core/inc/param_svc.h

```diff
@@ -426,6 +432,11 @@ typedef struct TagWelsSvcCodingParam: SEncParamExt {
       pSpatialLayer->uiTransferCharacteristics = pCodingParam.sSpatialLayers[iIdxSpatial].uiTransferCharacteristics;
       pSpatialLayer->uiColorMatrix =             pCodingParam.sSpatialLayers[iIdxSpatial].uiColorMatrix;
 
+      pSpatialLayer->bAspectRatioPresent = pCodingParam.sSpatialLayers[iIdxSpatial].bAspectRatioPresent;
+      pSpatialLayer->eAspectRatio = pCodingParam.sSpatialLayers[iIdxSpatial].eAspectRatio;
+      pSpatialLayer->sAspectRatioExtWidth = BIG_ENDIAN_16(pCodingParam.sSpatialLayers[iIdxSpatial].sAspectRatioExtWidth);
```

Comment:
The debugging showed that:
1) macro BIG_ENDIAN did nothing in my build. That's why result is the same with or without it.
2) ffmpeg's code:
```
static inline int decode_vui_parameters(H264Context *h, SPS *sps)
{
    int aspect_ratio_info_present_flag;
    unsigned int aspect_ratio_idc;

    aspect_ratio_info_present_flag = get_bits1(&h->gb);

    if (aspect_ratio_info_present_flag) {
        aspect_ratio_idc = get_bits(&h->gb, 8);
        if (aspect_ratio_idc == EXTENDED_SAR) {
            sps->sar.num = get_bits(&h->gb, 16);
            sps->sar.den = get_bits(&h->gb, 16);

```
doesn't swap bytes for sar.num and sar.den .
I looked into ISO/IEC 14496-10:2004(E) and ITU-T H.264 , and found nothing about endianness of numerical values (maybe it's just my bad watch)..
Looks like we have to trust ffmpeg, and remove BIG_ENDIAN_MACRO. I'll amend the commit.